lib/brains/forms.py

```diff
@@ -188,6 +190,75 @@ def clean(self):
         return self.cleaned_data
 
 
+class SaleForm(forms.Form):
+    amount = forms.DecimalField(
+        # Arbitrary values should they be set in a config somewhere?
```

Comment:
yes, I think the product should be configured with something like `allow_variable_amount=True` and this form should bail if that's not True when accepting a declared amount. Could be done in a follow-up patch. Otherwise we could some `0.01` purchases of fixed price products ;)